filters/tools.py

```diff
@@ -145,6 +147,13 @@ def layer_from_source(source_uri, name):
         elif source_uri.endswith(raster_extensions):
             qgis_layer = QgsRasterLayer(sanitized_uri, name)
 
+        elif source_uri.endswith(qlr_extensions):
+
+            qgis_layer = QgsMapLayer.fromLayerDefinitionFile(sanitized_uri)
+            if qgis_layer:
```

Comment:
Missing the `isValid()`?

Reply:
Thanks @Gustry ! `isValid()` is tested in the outside in `map_composition.py` together with the error message to return. In here, I just check if the loader returns empty list or with loaded layer. I'm still not sure how to handle multiple layer, though. Maybe this one will do for now?